src/options/smt_options.toml

```diff
@@ -342,6 +342,14 @@ header = "options/smt_options.h"
   type       = "bool"
   default    = "false"
   help       = "calculate sort inference of input problem, convert the input based on monotonic sorts"
+  
+[[option]]
+  name       = "symmetryBreaker"
+  category   = "regular"
+  long       = "symmetry-break"
```

Comment:
It might be more consistent to call the option `--symmetry-breaker`.

Reply:
Yes, this is a better name.

Reply:
I changed the name to --symmetry-breaker-exp for now because "--symmetry-breaker" has been taken.